src/main/scala/com/databricks/labs/overwatch/pipeline/SilverTransforms.scala

```diff
@@ -757,10 +757,11 @@ trait SilverTransforms extends SparkSessionWrapper {
           'spark_conf,
           'driver_instance_pool_id,
           'instance_pool_id,
-          'spark_version,
+          'effective_spark_version.alias("spark_version"),
           (unix_timestamp('Pipeline_SnapTS) * 1000).alias("timestamp"),
           'Pipeline_SnapTS.cast("date").alias("date"),
-          'creator_user_name.alias("createdBy")
+          'creator_user_name.alias("createdBy"),
+          'runtime_engine
```

Comment:
There are a lot of nulls at this point and as per the API docs, when it's null spark_version is used to determine the runtime_engine, so please add a rule to properly enhance the runtime_engine field when it is null.